docs/configure-custom-values-scalardb-cluster.md

```diff
@@ -6,13 +6,12 @@ This document explains how to create your custom values file for the ScalarDB Cl
 
 ### Image configurations
 
-You must set `scalardbCluster.image.repository` and `scalardbCluster.image.tag`. Please specify the container repository information that you will pull the ScalarDB Cluster container image from.
+You must set `scalardbCluster.image.repository`. Please specify the container repository information that you will pull the ScalarDB Cluster container image from.
```

Comment:
I removed the `tag` configuration. This is because we recommend using the default version that is set on the Helm Chart side.

However, users have to set `*.image.repository` configuration. This is because we provide the commercial license products on each Marketplace.

Reply:
As I mentioned in another comment, ScalarDB Cluster is a private (commercial license) product.
So, your assumption is correct. This configuration is required.
I will apply your suggestion.
Thank you!